api/database/migrations/2023_09_20_211150_combine_pool_skills.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        // create the combined table
+        Schema::create('pools_skills', function (Blueprint $table) {
```

Comment:
By convention in Laravel, pivot tables are named with the singular model names in alphabetical order.  I think this should be `pool_skill`.

Reply:
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7953/commits/901217cd56479c1d72a27cc9881cce032affa221

As you can tell from this PR alone, removing `pools_essential_skills` and `pools_nonessential_skills`, does not appear that convention has been followed much 😛

Reply:
Yeah, it's true.  :disappointed: Some have definitely slipped through.

Reply:
Ref: https://laravel.com/docs/10.x/eloquent-relationships#many-to-many-table-structure

Reply:
:+1: